public-dashboard-client/src/page-sentencing/PageSentencing.js

```diff
@@ -63,6 +81,22 @@ export default function PageSentencing() {
         locations: apiData.judicial_districts,
       },
     },
+    {
+      title: SECTION_TITLES[PATHS.sentencing].recidivism,
+      description: (
+        <>
+          After release from prison, a significant proportion of formerly
+          incarcerated folks end up back in prison. This is typically termed
+          “recidivism.” The below graph shows recidivism as reincarceration;
```

Comment:
Are these stylized quotes what we want instead of generic `"`? I'm always paranoid that characters like that won't render for properly for someone.

Reply:
In desktop publishing we used to call mangled text encodings "gremlins" and they were constantly sneaking into print. But I'm not really worried about this in a web browser context honestly? We are using them all over the place in this application and I'm not aware of any issues. I don't think any of the browsers we support are old enough to have those kinds of widespread Unicode support problems anymore.

I don't see how we could realistically maintain a policy like this without a lint rule, and I have not been able to find any existing rules or plugins that address this use case ... which suggests to me that it is not a thing we need to be worried about